dev/setup.sh

```
@@ -83,11 +90,15 @@ kubectl apply -f manifests/crd.yaml -n $FOURNOS_NAMESPACE
 # ---------------------------------------------------------------
 echo ""
 echo "Applying Fournos manifests..."
-for rbac_file in manifests/rbac/*.yaml; do
-  cat "$rbac_file" | NAMESPACE=$FOURNOS_NAMESPACE envsubst | kubectl apply -f- -n $FOURNOS_NAMESPACE
+kubectl apply -f manifests/rbac/sa_fournos.yaml -n "$CONTROLLER_NAMESPACE"
+kubectl apply -f manifests/rbac/sa_fournos.yaml -n "$FOURNOS_NAMESPACE"
+for rbac_file in manifests/rbac/role_fournos.yaml manifests/rbac/rolebinding_fournos.yaml; do
+  cat "$rbac_file" | CONTROLLER_NAMESPACE=$CONTROLLER_NAMESPACE envsubst '$CONTROLLER_NAMESPACE' | kubectl apply -f- -n $FOURNOS_NAMESPACE
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)